### PR TITLE
Added Contact us page

### LINKED ISSUE #246: contact us page in the Footer section
## Is your feature request related to a problem? Please describe.

A contact page form by which user can contact org or codeForCause team in the case of any query and if he/she wants to report any bug or problem etc. the form contains student name, email, text filed (in which he/she explains the issue), number optional, etc. 

Other than this on the contact form we can mention the organization's email address, content numbers a map that shows org HQ, organization social accounts, etc.

## Describe the solution you'd like
 
A separate page located under the profile footer section.

Cc/ @Abhishek-kumar09

### PR DESCRIPTION
**Pull request by Abhijay007**
**Issue Number**

fixes #246

**Describe the changes you've made**

Added contact us page and added links for the same under profile section footer

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**

`NA`

**Additional context (OPTIONAL)**

Screenshot 1          |  Screenshot 2             
:-------------------------:|:-------------------------:|
contact us page|contact-us_2

**checklist**

- [x] My code follows the code style of this project.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

**Provide a Deployed link of route/page that needs to review**

Preview: https://deploy-preview-262--festive-allen-a7f545.netlify.app/ContactUs

### CODE REVIEW

**Comment by sakshi-choudhary** [comment]
@Abhijay007 I think you should add `form validation` as well, you can make use of **_yup_** for this.

File: src/pages/ContactUs/contactUs.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    color:white !important;
+}
+
+.MuiFormLabel-root.Mui-focused{
+    color:white !important;
+}
+
+.MuiInputBase-input{
+    color:white !important;
+}
```
Comment (Abhishek-kumar09) [inline src/pages/ContactUs/contactUs.css:11]: Put it inside makeStyles

File: src/pages/ContactUs/index.js [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+        </Grid>
+        <Typography className={classes.title} variant="h2" color="secondary">
+          Connect with us :
+        </Typography>
+        <div className={classes.socialIcons}>
+          <FacebookIcon href="#" className={classes.socialIcons} />
+          <TwitterIcon href="#" className={classes.socialIcons} />
+          <InstagramIcon href="#" className={classes.socialIcons} />
+          <LinkedInIcon href="#" className={classes.socialIcons} />
+          <YouTubeIcon href="#" className={classes.socialIcons} />
```
Comment (Abhishek-kumar09) [inline src/pages/ContactUs/index.js:168]: Give codeforcause links here,

You can found all in codeforcause.org footer

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
Resolve conflicts

File: src/pages/ContactUs/index.js (hunk 2 above)
Comment (Abhijay007) [reply src/pages/ContactUs/index.js:168]: Ok, @Abhishek-kumar09 I will do all the requested changes :) 
I am also implementing the changes suggested by @sakshi-choudhary.

Thanks for reviewing my PR 👍

**Comment by Abhijay007** [comment]
@Abhishek-kumar09 can I work on this(#165) issue as it seems inactive?

**Pull request merged**